website/docs/getting-started/ui.mdx

```diff
@@ -109,4 +109,4 @@ Finally let's look at the Source in more detail - go back to the Details tab, an
 
 As with an Application detail view, you can see key information about how the resource is defined.
 
-Now we are familiar with the Dashboard, let's deploy a new application :sparkles:.
+Now that we are familiar with the WGE dashboard, let's deploy a new application :sparkles:.
```

Comment:
```suggestion
Now that we are familiar with the dashboard, let's deploy a new application :sparkles:.
```

Although the description applies to both OSS and WGE dashboards, the screenshots we use are from OSS.